package com.medicine.backend.repositories;

import com.medicine.backend.models.Application;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.UUID;

@Repository
public interface ApplicationRepository extends JpaRepository<Application, UUID> {
    List<Application> findAllByStudentId(UUID studentId);
    List<Application> findAllByHospitalServiceHospitalId(UUID hospitalId);
}
